Source/WebCore/html/canvas/CanvasRenderingContext.h

```diff
@@ -49,7 +49,7 @@ class SVGImageElement;
 class WebGLObject;
 enum class PixelFormat : uint8_t;
 
-class CanvasRenderingContext : public ScriptWrappable {
+class CanvasRenderingContext : public ScriptWrappable, public CanMakeWeakPtr<CanvasRenderingContext>, public CanMakeCheckedPtr {
```

Comment:
Should include <wtf/CheckedRef.h> since this is where CanMakeCheckedPtr is defined.